SCClassLibrary/Common/Control/GraphBuilder.sc

```diff
@@ -121,11 +121,11 @@ NamedControl {
 	}
 
 	init {
-		var prefix, ctlName, ctl, selector;
+		var prefix, str;
 
 		name !? {
-			name = name.asString;
-			if(name[1] == $_) { prefix = name[0]; ctlName = name[2..] } { ctlName = name };
+			str = name.asString;
+			if(str[1] == $_) { prefix = str[0] };
 		};
```

Comment:
Not removing the prefix from the ctlName might result in some ambiguity. Just make sure to ckeck whether something like `\t_trig.kr` and `trig.tr` don't interfere in a bad way...

Reply:
> Not removing the prefix from the ctlName might result in some ambiguity. Just make sure to ckeck whether something like \t_trig.kr and trig.tr don't interfere in a bad way...

I had the same question, but... in the currently released version, ctlName is assigned in this block and then never used.

https://github.com/supercollider/supercollider/blob/develop/SCClassLibrary/Common/Control/GraphBuilder.sc#L123

Here, there are exactly 3 occurrences of ctlName: once to declare it, and twice in the `if` where it's assigned, and then... nothing.

So ctlName is cruft, and deleting the variable entirely is correct.